classes/plugins/DOIPubIdExportPlugin.php

```diff
@@ -148,34 +155,39 @@ public function getPublishedSubmissions($submissionIds, $context)
      */
     public function getPublishedIssues($issueIds, $context)
     {
-        $publishedIssues = [];
-        foreach ($issueIds as $issueId) {
-            $publishedIssue = Repo::issue()->get($issueId);
-            $publishedIssue = $publishedIssue->getJournalId() == $context->getId() ? $publishedIssue : null;
-            if ($publishedIssue && $publishedIssue->getStoredPubId('doi')) {
-                $publishedIssues[] = $publishedIssue;
-            }
-        }
+        $publishedIssues = Repo::issue()
```

Comment:
This could be returned directly instead of being assigned to the intermediate `$publishedIssues`.